changelog.dd

```diff
@@ -45,6 +45,8 @@ $(BUGSTITLE Library Changes,
         $(XREF conv, to) instead.)
     $(LI $(RELATIVE_LINK2 min-max-element,
     `std.algorithm.searching.{min,max}Element` for ranges have been added.))
+    $(LI $(RELATIVE_LINK2 quantize, Added $(XREF math, quantize), for rounding to
```

Comment:
Don't put XREF in the summary line. The whole line becomes a link, and XREF becomes a link, and then things get messed up.